app/controllers/api/v1/targets_controller.rb

```diff
@@ -7,6 +7,10 @@ def create
         @target.save!
       end
 
+      def index
+        @targets = policy_scope(Target)
```

Comment:
I think that targets belongs to the users, so here you should only sent logged user targets, wdyt?

Reply:
I added that dependency on a feature branch